content/en/docs/reference/node/node-status.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Node Status
+weight: 80
+---
+
```

Comment:
Please write a brief introduction. The introduction should link to https://kubernetes.io/docs/concepts/architecture/nodes/